spec/sober_swag/compiler/type_spec.rb

```diff
@@ -183,7 +183,7 @@ def self.compiling_output(&block)
   end
 
   context 'with a class that has nested things' do
-    ExampleInput = SoberSwag.input_object do # rubocop:disable RSpec/LeakyConstantDeclaration
+    ExampleInput = SoberSwag.input_object do # rubocop:disable RSpec/LeakyConstantDeclaration, Lint/ConstantDefinitionInBlock
```

Comment:
Multiple lines to keep under 80 characters.

```suggestion
    # rubocop:disable RSpec/LeakyConstantDeclaration
    # rubocop:disable Lint/ConstantDefinitionInBlock
    ExampleInput = SoberSwag.input_object do
```

Reply:
This repo is configured for 160 character lines, and I would have to add `rubocop:enable` comments after this, which... I don't feel like doing.